packages/server/src/synchronizer/pipeline/rules/relative/relative-paths.test.ts

```diff
@@ -1,38 +1,51 @@
 import {relativeToAbsolute, replaceRelativeImports} from './index'
-import {normalize} from 'path'
 
 describe('relativeToAbsolute', () => {
+  // I don't like this one bit, but Node's path module behaves so differently
+  // based on the runtime platform that it's just easier to test paths as they
+  // would be on the platform currently executing the test suite. Tests should
+  // be run on both platforms, so both test paths will be executed - just not
+  // during the same run.
+  const isWindows = process.platform === 'win32'
   const tests = [
     {
       name: 'Provides an absolute path within app',
       input: {
-        relativeImport: normalize('../components/three'),
-        filename: normalize('/projects/blitz/blitz/app/users/pages.ts'),
+        relativeImport: '../components/three',
+        filename: isWindows
+          ? 'C:\\projects\\blitz\\blitz\\app\\users\\pages.ts'
```

Comment:
Maybe could make a function to do this so we don't get so much testing noise? 

```ts
const platformSensitiveAbsolutePath = p => process.platform === 'win32' ?  'C:' + p.split('/').join('\\') : p
```

Reply:
Done!